src/sass/rx/partials/_rx-prescriptions.scss

```diff
@@ -41,7 +41,6 @@
 .rx-prescription-title {
   color: $color-primary;
   font-size: 1.4em;
-  line-height: 1;
```

Comment:
Minor side fix for prescription names in the cards getting cut off because their line-heights were too small.